en/docs/Administer/ProductConfigurations/changing-the-default-transport.md

```diff
@@ -147,13 +142,18 @@ The HTTP-NIO transport is a module of the Apache Synapse project. Apache Synapse
 </tbody>
 </table>
 
-### Connection throttling
+### Connection Throttling
+
+With the HTTP PassThrough, you can enable connection throttling to restrict the number of simultaneous open connections. To enable connection throttling, edit the `<PRODUCT_HOME>/repository/conf/deployment.toml` and add the following configuration under passthough_http configurations.
```

Comment:
Shall we change [1] to [2]

[1] To enable connection throttling, edit the `<PRODUCT_HOME>/repository/conf/deployment.toml` and add the following configuration under passthough_http configurations.

[2] To enable connection throttling, edit the `<API-M_HOME>/repository/conf/deployment.toml` and add the following configuration under `passthough_http` configurations.

Reply:
fixed with https://github.com/wso2/docs-apim/pull/540/commits/66d4d47c3b663dccc4ce708bc0ae2f9cf1e0ddff